src/server/storage/storageAPI.js

```diff
@@ -96,7 +96,7 @@ const setup = (app: Router, gunPublic: StorageAPI, storage: StorageAPI) => {
             if (isWhitelisted === false) throw new Error('Failed whitelisting user')
           })
           .catch(e => {
-            logger.error('addUserToWhiteList failed', { e, errMessage: e.message, userRecord })
+            logger.error('addUserToWhiteList failed', e.message, e, { userRecord })
```

Comment:
why it's needed ?

Reply:
That's the convention of error log arguments that we discussed and decided to follow with the team and Hadar earlier.
```
log.error(generalMessage, errorObj.message, errorObj, { some context data here }) 
```